  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\x07pi\\docs\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\x07pi\\docs\\pdf'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\x07pi\\docs\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\x07pi\\docs\\pdf'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\x07pi\\docs\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\x07pi\\docs\\pdf'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\\docs\x07pi\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\\docs\x07pi\\pdf'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\\docs\x07pi_docs\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\\docs\x07pi_docs\\pdf'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\\docs\x07pi\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 22] Invalid argument: '.\\docs\x07pi\\pdf'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: '.\\docs\\libs\\html'
  [...............................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 491, in main
    write_latex(docindex, options, options.action)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 563, in write_latex
    latex_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 146, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\latex.py", line 210, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: '.\\docs\\libs\\pdf'
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  Error: Could not find a file or object named \Lib
  Error: Nothing left to document!
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target docs/pdf\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target docs/pdf\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target docs\\pdf\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target docs/pdf\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 17]: 'target docs/pdf\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target docs/pdf\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target docs\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target /docs\n'
usage: epydoc.py ACTION [options] NAMES...

epydoc.py: error: Error reading config file .\config\epydoc_config.ini:
    File contains parsing errors: .\config\epydoc_config.ini
	[line 20]: 'target ./docs\n'
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: 'docs/api/html'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: 'docs/api/html'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: 'docs/api/html'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: 'docs/api/html'
  [.................................Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 489, in main
    write_html(docindex, options)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 522, in write_html
    html_writer.write(options.target)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 500, in write
    self._mkdir(directory)
  File "C:\Python24\Lib\site-packages\epydoc\docwriter\html.py", line 631, in _mkdir
    os.mkdir(directory)
OSError: [Errno 2] No such file or directory: 'docs/api/html'
  [.........................................................................]
  [.........................................................................]
  [.........................................................................]
  [.........Traceback (most recent call last):
  File "c:\python24\scripts\epydoc.py", line 16, in ?
    cli()
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 663, in cli
    main(options, names)
  File "C:\Python24\lib\site-packages\epydoc\cli.py", line 466, in main
    add_submodules=(options.action!='text'))
  File "C:\Python24\Lib\site-packages\epydoc\docbuilder.py", line 130, in build_doc_index
    doc_pairs = _get_docs_from_items(items, introspect, parse, add_submodules)
  File "C:\Python24\Lib\site-packages\epydoc\docbuilder.py", line 274, in _get_docs_from_items
    doc_pairs += _get_docs_from_submodules(
  File "C:\Python24\Lib\site-packages\epydoc\docbuilder.py", line 458, in _get_docs_from_submodules
    d = _get_docs_from_module_file(
  File "C:\Python24\Lib\site-packages\epydoc\docbuilder.py", line 411, in _get_docs_from_module_file
    parse_doc = parse_docs(
  File "C:\Python24\Lib\site-packages\epydoc\docparser.py", line 295, in parse_docs
    process_file(module_doc)
  File "C:\Python24\Lib\site-packages\epydoc\docparser.py", line 550, in process_file
    for toktype, toktext, (srow,scol), (erow,ecol), line_str in tok_iter:
  File "C:\Python24\lib\tokenize.py", line 229, in generate_tokens
    raise IndentationError(
IndentationError: unindent does not match any outer indentation level
  [.........................................................................]
